salt/modules/slack_notify.py

```diff
@@ -238,18 +238,24 @@ def post_message(channel,
 
 
 def call_hook(message,
+              channel=None,
```

Comment:
Can you move all of these new kwargs to the end of the list? The way this is now constitutes an API change, which we would like to avoid. :)